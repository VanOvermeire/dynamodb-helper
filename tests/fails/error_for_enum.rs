use dynamodb_helper::DynamoDb;

#[derive(DynamoDb)]
pub enum SomeEnum {
    Nested(String)
}

fn main() {}
