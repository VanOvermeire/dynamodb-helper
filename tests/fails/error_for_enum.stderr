error: DynamoDB macro cannot be used with an enum - use a struct instead
 --> tests/fails/error_for_enum.rs:4:10
  |
4 | pub enum SomeEnum {
  |          ^^^^^^^^
